google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineAssetProvider.java

```diff
@@ -44,4 +46,25 @@ public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Ar
 
     return null;
   }
+
+  @Nullable
+  @Override
+  public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Module module) {
+    VirtualFile[] contentRoots = ModuleRootManager.getInstance(module).getContentRoots();
+
+    for (VirtualFile contentRoot : contentRoots) {
+      // TODO clean this up - do we need something more robust?
+      VirtualFile descriptorFile
+          = contentRoot.findFileByRelativePath("web/WEB-INF/appengine-web.xml");
```

Comment:
also I'm pretty sure there is an API for finding a file. @joaoandremartins used it for a cloud debugger bug fix way back.